.github/workflows/test.yaml

```diff
@@ -31,7 +31,7 @@ jobs:
         with:
           environment-file: environment_cpu.yaml
 
-      # Setup Qt libraries for GUI testing
+      # Setup Qt libraries for GUI testing on Linux
```

Comment:
This line is unrelated to the specific issue being solved, but I've improved the comment anyway. Mostly because I forgot this, so next time it will be more obvious now that it is only relevant on Linux.